scripts/nodeinstaller.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Hi @tudorpintea999 👋 ! Thank you for the PR.

> To run the node by one command
> To expand to more non technical people to run locally a node

I believe we already have 2 ways of addressing this outside a `.sh` script:

1. `npx zksync-cli dev start`
2. `yarn hardhat node-zksync`

With those in mind, I'm unsure this script is necessary as it's also specific to Linux and tied down to `v0.1.0-alpha.6` of `era-test-node`

Reply:
Hello @MexicanAce i did not knew about that possiblity, then you re probably right, thanks !